docs/guides/integration-prebidjs.md

```diff
@@ -0,0 +1,193 @@
+
+:::tip
+Only root-level domains are required for account setup. For example, if you are going to use UID2 with Prebid.js on example.com and shop.example.com, you only need to provide the domain name example.com.
+:::
+
+## Add Prebid.js to your site
+
+Follow the [Prebid.js documentation](https://docs.prebid.org/dev-docs/getting-started.html) to add Prebid.js to your site. Make sure to use Prebid.js version **TODO** or higher.
+
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named "Unified ID 2.0". The module is listed under the section *User ID Modules*.
+
+Once you have added Prebid.js to your site and confirmed that it is working properly, you are ready to configure the UID2 module.
+
+:::tip
+Ensure the UID2 module is installed by finding the string "uid2IdSystem" in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
+
+## Configure the UID2 module
+
+To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** obtained during account setup, as well as the user's [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number).
+
+The following sections demonstrate the different ways to configure the UID2 module, as well as the requirements for the DII passed to the module.
+
+:::info
+The user's DII can be passed to the UID2 module hashed or unhashed. If the DII is passed to the module hashed, it must be normalized before hashing.
+
+See [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for more details.
+:::
+
+:::tip
+The UID2 module will ensure that all DII is hashed and encrypted before sending it to the UID2 service.
+:::
+
+### Configure for email
+
+Configure the UID2 module with an email address:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        email: 'user@example.com',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will normalize and hash the email address before sending the encrypted hash to the UID2 service.
+
+### Configure for phone
+
+Configure the UID2 module with a phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phone: '+1111111111',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will hash the phone number before sending the encrypted hash to the UID2 service.
+
+The publisher is responsible for normalizing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md#phone-number-normalization) for details on phone number normalization.
```

Comment:
probably need to bold the **The publisher is responsible for normalizing the phone number.** line